src/commands/goto.ts

```diff
@@ -108,11 +96,11 @@ function executeGotoLine(editor: vscode.TextEditor, count: number, extend: boole
     },
     'default': function (x: vscode.Selection) {
       return 0
-    }
+    },
   }
   
   editor.selections = editor.selections.map(x => {
-      const npos:vscode.Position = new vscode.Position(x.active.line,  (positions[position] || positions['default'])(x));
+      const npos = new vscode.Position(x.active.line,  (positions[position] || positions['default'])(x));
```

Comment:
If you're type-checking this file using TypeScript, you should notice that you no longer need ` || positions['default']`, because by changing `position`'s and `positions`'s types, you ensured that `positions[position]` would never be `undefined`, therefore having a default value is unneeded.

Also, there are two spaces after that comma, rather than one ;)

Reply:
Same remark for line 128.

Reply:
Thank you for reviewing. It's much more detailed than I am used to, but that's totally fine, helps me alot :)
Do you use any tools for checking the conventions?

Reply:
I have another branch where I implemented a revealCursor mechanisms to make the editor change the visible lines on movement.